### PR TITLE
Milestone 3: Deploy page

### PR DESCRIPTION
**Pull request by isume295**
in this milestone I:

- added alt to the img tag
- deployed the website

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

You have submitted a project, that has not following correct flow for asking review. Kindly check the project requirements.
Screenshot 2023-04-22 at 3 42 26 PM


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- walkthrough -->
+
+##  Walkthrough <a name="walkthrough"></a>
+
+Feel free to check the [walkthrough video page](https://www.loom.com/share/3a1ed8cf22274d2f8a8cc7d0b886a679).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (amon-cofie) [inline README.md]: [OPTIONAL] You can move this walkthrough presentation video to the live demo section as it counts as a demo of your project

**Review by amon-cofie — CHANGES_REQUESTED** [review]
Hi @isume295 , 👋 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there! 💪 




## Suggested changes

- [ ] You spoke professionally in your video presentation and you mentioned the reason why built the project and some challenges that you faced and that's good 👍. However, I suggest that you also show your code base and talk about a piece of the code that you are most proud of. Also, while presenting the preview of the project, it would be nice to see a demonstration of the responsiveness of the page 😉 

- [ ] In your program outline section, you used divs for each of the items on the list and I want to suggest that you use article tags instead. This is more semantic and can improve the SEO scores of your page. There's a screenshot below to give you context to what I mean and you can find out more about the article tag [here](https://www.w3schools.com/tags/tag_article.asp) 
image



_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @isume295, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**